### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by zuphilip**
I implemented a `Dockerfile` and put an [image on hub.docker](https://hub.docker.com/r/zuphilip/translation-server/). This enables you to run the `translation-server` as a docker container, e.g.

```
$ docker run -d -p 1969:1969 --name zts-container 'zuphilip/translation-server:docker'
```

Moreover, it is possible to build the container within the directory with

```
$ docker build -t zts -f Dockerfile .
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# USAGE:
+# $ docker build -t zts -f Dockerfile .
+# $ docker run -d --rm --port 1969:1969 --name zts-container zts
+# 
+
+FROM ubuntu:14.04
+
+RUN apt-get update && apt-get install -y curl git firefox
+
+ENV VERSION 41.0
```
Comment (dstillman) [inline Dockerfile]: Should be 41.0.2 (though really we should update the build script to get it working with Firefox 45 ESR).

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ENV ARCHITECTURE linux-x86_64
+# alternatively linux-i6841
```
Comment (dstillman) [inline Dockerfile]: linux-i686

**Comment by kba** [comment]
I've reduced the image to ~320 MB and made it an automatic build to be rebuilt when the translation-server (or any translators or zotero itself) are updated. See https://hub.docker.com/r/kbai/zotero-translation-server/

**Comment by zuphilip** [comment]
Okay, my PR is merged now. Let us see how to continue:
- Dan: would you like to create a docker login for zotero and distribute the official `translation-server` at some adress like https://hub.docker.com/r/zotero/translation-server ? A webhook from the github repo here would make sense to automatically have always the newest docker container. If I understood @kba correctly you could then even trigger that image to rebuild automatically on new commits to the zotero or commits to translators repo. Moreover, might in the future also make sense to provide some docker file for the `data-server`...
- Konstantin: Do you want to make a new PR now here with your improvements? The smaller size looks really good and more automatism sound also fine (also I am not sure that this deletion https://github.com/kba/translation-server/commit/ce46a798be69fee78482179f6680db34a485e0b1#diff-5e7263385927431c0b62df682337b5f8L1 is intended, but I guess it is easier to discuss in a new PR).
- I will try to improve the `README.md` and also mention the docker image which we have now.

**Pull request merged**